### PR TITLE
metrics (ticdc): add some changefeed releated metrics panel in Grafana

### LINKED ISSUE #6937: Add some metrics to reflect exception time of a changefeed to catch up upstream TiDB.
### Is your feature request related to a problem?

For now, TiCDC use the derivative of checkpointTs to calculate the exception time of a changefeed to catch up upstream TiDB, it can sharp rise to a very large value if the changfeed's checkpointTs stucks -- which can happen when one table's checkpointTs stucks but other table's checkpointTs do not. This phenomenon may confuses users.

### Describe the feature you'd like

1.  Add a metrics to monitor the `changed events remain in cdc`. 
2. Use `changed events remain in cdc / sink speed` to calculate the ETA.

### Describe alternatives you've considered

_No response_

### Teachability, Documentation, Adoption, Migration Strategy

_No response_

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #6937 

#### Tests 

 - Manual test (add detailed scripts or steps below)
image

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/processor/pipeline/sorter.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func createSorter(ctx pipeline.NodeContext, tableName string, tableID model.Tabl
 			dbActorID := ssystem.DBActorID(uint64(tableID))
 			compactScheduler := ctx.GlobalVars().SorterSystem.CompactScheduler()
 			levelSorter, err := db.NewSorter(
-				ctx, tableID, startTs, ssystem.DBRouter, dbActorID,
+				ctx, ctx.ChangefeedVars().ID, tableID, startTs, ssystem.DBRouter, dbActorID,
```
Comment (3AceShowHand) [inline cdc/processor/pipeline/sorter.go:123]: please do not use `ctx` to pass changefeed information.

**Review by overvenus — APPROVED** [review]

**Review by nongfushanquan — APPROVED** [review]

File: cdc/processor/pipeline/sorter.go (hunk 1 above)
Comment (asddongmen) [reply cdc/processor/pipeline/sorter.go:125]: I will refine this in next PR.

**Pull request merged**